src/components/Text/Text.tsx

```diff
@@ -1,37 +1,31 @@
 import * as React from 'react';
 import { Text as ChakraText, BoxProps } from '@chakra-ui/core';
 
-export enum TextVariants {
-  CONTENT = 'CONTENT',
-  CONTENTBOLD = 'CONTENTBOLD',
-  SUBCONTENT = 'SUBCONTENT',
-  SUBCONTENTBOLD = 'SUBCONTENTBOLD',
+export enum TextWeights {
+  normal = 'normal',
+  bold = 'bold',
 }
 
-export type TextProps = BoxProps & {
-  variant?: TextVariants;
-};
-
-const fontSizeByVariant = {
-  [TextVariants.CONTENT]: 'm',
-  [TextVariants.CONTENTBOLD]: 'm',
-  [TextVariants.SUBCONTENT]: 's',
-  [TextVariants.SUBCONTENTBOLD]: 's',
-};
+export enum TextSizes {
+  normal = 'normal',
```

Comment:
What about simplifying this and using a boolen for `bold`? Like `<Text bold />`. If bold is not present then it's normal. I used to have that  in my previous design system and it was nice

Reply:
Mh 🤔 
What if we add more weights later on?
Would you do the same for the size?

This is something we don't do on the other components (like in `Card` for example, we do `size="small"` and not `small`

IMO it's safer and more clear to go for an enum, rather than a prop
How would you expect the component to behave if we do something like `<Text bold light />`?

Any opinion @heetch/frontend?

Reply:
I prefer enums for that, I wouldn't want to define a new prop every time we have a new weight

Reply:
Works for you @obedparla?

Reply:
sounds good to me, I wanted to pur the idea out there 👍